desk/lib/bolt.hoon

```diff
@@ -6,24 +6,23 @@
 ::  ...
 ::  %-(agent:bolt your-agent)
 ::
-
 /-  *bolt
 /+  verb, agentio
-
+::
 =>
-|% 
-+$  card  card:agent:gall
-+$  versioned-state
-  $%  state-0
-  ==
-+$  state-0  
-  $:  %0 
-      which=?
-      kids=?
-      =blacklist 
-      =whitelist
-  ==
---
+  |% 
```

Comment:
Whether this is indented is a matter of style but for these very small cores I personally prefer it.